.circleci/config.yml

```diff
@@ -2,18 +2,18 @@ version: 2.1
 jobs:
   lint:
     docker:
-      - image: salsify/ruby_ci:2.6.10
+      - image: ruby:3.0.6
```

Comment:
We lint against the oldest supported version of Ruby. Can you get the lint job working against `ruby:2.6.10`?